package.json

```diff
@@ -325,6 +325,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-dropzone": "^4.2.5",
+    "react-focus-lock": "^2.5.0",
```

Comment:
What is this for?

Reply:
The `MegaMenu` [uses it](https://github.com/department-of-veterans-affairs/vets-website/blob/bb3e023828eac88dfc686fad59b9bd0ffc3df166/src/platform/site-wide/mega-menu/components/SubMenu.jsx#L3). It's unclear to me why this was in `node_modules` before but isn't after this change, though. At any rate, it's a direct dependency of `vets-website`, so it should be in the dependency list here now. :man_shrugging: